components/Home/sidebar/api/fetchOpenBounties.ts

```diff
@@ -0,0 +1,21 @@
+import API from "~/config/api";
+import { sendAmpEvent } from "~/config/fetch";
+import { buildApiUri } from "~/config/utils/buildApiUri";
+import { Helpers } from "@quantfive/js-web-config";
+
+type Args = {
+  onError: (error: Error) => void;
+  onSuccess: (response: any) => void;
+  page?: number;
+};
+
+export const fetchOpenBounties = ({ onError, onSuccess, page }: Args): void => {
```

Comment:
Can we put this one in `Bounty/api` folder?

I want to keep as much of bounty related things in the bounty modules so it is obvious where to look in.
Also, will likely be used outside of homepage

Reply:
sure